config_defaults/subtests/docker_cli/build.ini

```diff
@@ -76,9 +76,9 @@ __example__ = dockerfile_dir_path, postproc_cmd_csv
 use_config_repo = no
 dockerfile_dir_path = https://raw.githubusercontent.com/autotest/autotest-docker/master/subtests/docker_cli/build/full/Dockerfile
 # Hard code this to base-image used in remote dockerfile (above)
-docker_registry_host = docker.io
-docker_registry_user = stackbrew
-docker_repo_name = centos
+docker_registry_host = registry.access.redhat.com
+docker_registry_user = rhel7
+docker_repo_name = rhel
```

Comment:
IIRC, I disabled the images count check on both ``git_path`` and ``https_path``.  Also, I'm working on testing a fix for the "fedora base image is missing python" problem.